### ISSUE TITLE
[TEST] rocky linux pipeline not working due to "accept terms and subscribe"

### ISSUE THREAD

**Issue by chriscchien** [longhorn/longhorn#7647]
## What's the test to develop? Please describe

rocky linux [AMD64](https://ci.longhorn.io/job/public/job/master/job/rockylinux/job/amd64/job/longhorn-tests-rockylinux-amd64/26/console), [ARM64](https://ci.longhorn.io/job/public/job/master/job/rockylinux/job/arm64/job/longhorn-tests-rockylinux-arm64/5/) pipeline create instance error due to
```
Error: Error launching source instance: OptInRequired: In order to use this AWS Marketplace product you need to accept terms and subscribe. To do so please visit https://aws.amazon.com/marketplace/pp?sku=by3nk5xsnprhdxayivcnsk1ol
	status code: 401, request id: b26e2525-9873-4542-a906-a57f57085c35

  with aws_instance.lh_aws_instance_controlplane_k3s[0],
  on k3s_instances.tf line 2, in resource "aws_instance" "lh_aws_instance_controlplane_k3s":
   2: resource "aws_instance" "lh_aws_instance_controlplane_k3s" {
```

We have a release test [item](https://app.qase.io/run/LH/dashboard/5/af2f9aeaa8cdf64e56502e856959f45774dc10fa) releated to this distro, create this ticket for tracking, thank you.

cc @longhorn/qa 

## Describe the tasks for the test

N/A

## Additional context

N/A

**Comment by innobead** [comment]
@yangchiu please help with tihs.

**Comment by yangchiu** [comment]
@innobead I am unable to do that:
```
The seller has not made this product available in the geo location associated with your customer account.
```

**Comment by innobead** [comment]
Thanks @yangchiu . Then, let's test the original version for this reason. 

cc @khushboo-rancher needs to update the doc again for this limitation.

**Comment by chriscchien** [comment]
We can use rocky linux to run e2e test on pipeline [AMD64](https://ci.longhorn.io/job/public/job/master/job/rockylinux/job/amd64/job/longhorn-tests-rockylinux-amd64/27/), [ARM64](https://ci.longhorn.io/job/public/job/master/job/rockylinux/job/arm64/job/longhorn-tests-rockylinux-arm64/11/)

**Comment by innobead** [comment]
@chriscchien Did u mean it's working now to test 9.3?

**Comment by chriscchien** [comment]
> @chriscchien Did u mean it's working now to test 9.3?

Yes when image=9.2, it need subscribe but use 9.3 as instance image the test is running properly([AMD](https://ci.longhorn.io/job/public/job/master/job/rockylinux/job/amd64/job/longhorn-tests-rockylinux-amd64/31/), [ARM](https://ci.longhorn.io/job/public/job/master/job/rockylinux/job/arm64/job/longhorn-tests-rockylinux-arm64/14/)), I will close this ticket once the test complete.

**Comment by innobead** [comment]
@chriscchien as I mentioned, please help update the pipeline with Rocky 9.3 instead.
